stake-pool/cli/src/main.rs

```diff
@@ -621,36 +644,41 @@ fn command_deposit(
         &mut total_rent_free_balances,
     ));
 
+    let referrer = referrer.unwrap_or(stake_pool.manager_fee_account);
```

Comment:
We :heart: CLIs?
```suggestion
    let referrer = referrer.unwrap_or(token_receiver);
```

Reply:
Ok, user deserves credit for using the cli :P